lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -179,7 +188,8 @@ def response(action, succeeded, response)
           error_code: error_code_from(succeeded, response),
           test: test?,
           avs_result: avs_result,
-          cvv_result: cvv_result
+          cvv_result: cvv_result,
+          use_network_token: use_network_token
```

Comment:
@tanyajajodia @shasum This indicates weather NT was used for the transaction. An open question is weather we also want to include a separate field that indicates whether NT _was supposed to be used_

Reply:
@arbianchi Is this needed, when caller knows it passed in a `NetworkTokenizationCreditCard` as the payment method.

Reply:
I'm not sure if need these attributes in the AM side of things. Probably just tracking them in Core?